rasa/cli/interactive.py

```diff
@@ -1,20 +1,16 @@
 import argparse
+import asyncio
 import os
-from typing import List, Text
+from typing import List, Optional, Text
 
+from rasa.cli import utils
 import rasa.cli.train as train
 from rasa.cli.arguments import interactive as arguments
 from rasa import data, model
 
-
 # noinspection PyProtectedMember
-from rasa.cli.utils import get_validated_path, print_error
-from rasa.constants import (
-    DEFAULT_DATA_PATH,
-    DEFAULT_MODELS_PATH,
-    DEFAULT_ENDPOINTS_PATH,
-)
-from rasa.model import get_latest_model
+from rasa.constants import DEFAULT_MODELS_PATH, DEFAULT_ENDPOINTS_PATH
+from rasa.importers.rasa import TrainingDataImporter
```

Comment:
not sure why this isn't failing :-D 
```suggestion
from rasa.importers.rasa importer TrainingDataImporter
```

Reply:
whoops, switched that one over wrong -- I guess because the rasa one also imports trainingdataimporter? 🤔 good catch!